src/Routes.js

```diff
@@ -56,7 +56,7 @@ const Routes = (props) => (
     <Route path="/home_dac_info" component={HomeDacInfo} />
     <Route path="/home_researcher_info" component={HomeResearcherInfo} />
     <Route path="/election404" component={Election404} />
-    <Route path="/NIHpilotinfo" component={NIHpilotinfo} />
+    <Route path="/nih_pilot_info" render={(routeProps) => <NIHPilotInfo {...routeProps} {...props} />} />
```

Comment:
The NIHPilotInfo Component doesn't use any prop variables so you don't need to pass in props to the component.